examples/autocomplete.amp.html

```diff
@@ -37,12 +40,88 @@
       color: red;
       text-decoration: underline;
     }
+    .wide {
+      width: 50vw;
+    }
+    .autocomplete-selected {
+      color: blue;
+      text-decoration: underline;
+    }
+    .profile-pic {
+        width: 30px;
+        height: 30px;
+        margin-right: 10px;
+        margin-left: -5px;
+    }
+    .close-friends {
+      display: flex;
+      align-items: center;
+      line-height: 18px;
+    }
   </style>
 </head>
 <body>
-    <h1>Autocomplete</h1>
+  <h1>Autocomplete</h1>
+  <h2>Inline</h2>
+    <form method="post" action-xhr="/form/echo-json/post" target="_blank">
+      <amp-autocomplete inline="@" min-characters="0" suggest-first filter="none" 
```

Comment:
Can you [run this file through prettier](https://prettier.io/playground/)? But leave the AMP boilerplate as the one-line sausage that it is.